internal/utils/utils.go

```diff
@@ -15,6 +15,7 @@ import (
 	configv1 "github.com/openshift/api/config/v1"
 	v1 "k8s.io/api/core/v1"
```

Comment:
Since we touch this file, let's consider to use proper convention-safe alias here `corev1` instead of simply `v1`.